kuma/users/views.py

```diff
@@ -138,6 +141,22 @@ def ban_user_and_cleanup_summary(request, user_id):
     else:
         user_ban.update(by=request.user, reason='Spam')
 
+    # The revisions to be reverted
+    revisions_to_be_reverted = user.created_revisions\
+        .filter(id__in=request.POST.getlist('revision-id'))\
+        .prefetch_related('document').defer('content', 'summary')\
+        .order_by('-created')
+
+    # Submit revisions to Akismet as spam
+    for revision in revisions_to_be_reverted:
+        submission = RevisionAkismetSubmission(sender=request.user, type="spam")
+        akismet_submission_data = {'revision': revision.id}
+
+        data = RevisionAkismetSubmissionSpamForm(data=akismet_submission_data, instance=submission, request=request)
+        # TODO: What to do of data is not valid? Currently, fail silently
+        if data.is_valid():
+            data.save()
+
```

Comment:
We should call this out as requiring further action on the summary page, which is needed for later phases anyway.  After our discussion during the dev meeting, I agree that this work should be done in this PR.